### PR TITLE
Filter dropdown

### LINKED ISSUE #150: Filterbar dropdown menu bug
Fix filter bar dropdown bug.

### PR DESCRIPTION
**Pull request by JayDampitan**
## Changes
1. Fix bug on filter bar dropdown. 

Closes #150

### CODE REVIEW

**Review by gonzalespaulb — APPROVED** [review]
Easy peasy

File: pokeapp/src/components/FilterDropDown.js [hunk 1]
```diff
@@ -186,7 +186,8 @@ const FilterDropDown = ({
             className={
               isOption1 ? "sort-option-btn option-clicked" : "sort-option-btn"
             }
-            onClick={() => {
+            onClick={(event) => {
+              event.stopPropagation();
```
Comment (BlackCubes) [inline pokeapp/src/components/FilterDropDown.js:190]: Good catch

**Review by BlackCubes — APPROVED** [review]
Good catch on the bug!

**Review by Lap343 — APPROVED** [review]
Nice job

File: pokeapp/src/components/FilterDropDown.js (hunk 1 above)
Comment (JorgeLVilla) [reply pokeapp/src/components/FilterDropDown.js:190]: Noice!

**Review by JorgeLVilla — APPROVED** [review]
Good job Jay!

**Pull request merged**